app/views/spree/admin/variants/_form.html.haml

```diff
@@ -1,10 +1,10 @@
 .label-block.left.six.columns.alpha{'ng-app' => 'admin.products'}
   .field
     = f.label :display_name, t('.display_name')
-    = f.text_field :display_name, class: "fullwidth"
+    = f.text_field :display_name, class: "fullwidth", placeholder: t('admin.products.display_name_placeholder')
```

Comment:
you should use relative paths, this should be
 placeholder: t('.display_name_placeholder')

and then you need to add the keys in en.yml in line 3381 and 3382, which is path spree.admin.variants.form (the path of this haml file)

Reply:
Just made these changes.